pyiron/sphinx/base.py

```diff
@@ -42,36 +42,43 @@
 s = Settings()
 
 BOHR_TO_ANGSTROM = (
-    scipy.constants.physical_constants["Bohr radius"][0] / scipy.constants.angstrom
+    scipy.constants.physical_constants["Bohr radius"][0] /
+    scipy.constants.angstrom
 )
 HARTREE_TO_EV = scipy.constants.physical_constants["Hartree energy in eV"][0]
 HARTREE_OVER_BOHR_TO_EV_OVER_ANGSTROM = HARTREE_TO_EV / BOHR_TO_ANGSTROM
 
 
 class SphinxBase(GenericDFTJob):
     """
-    Class to setup and run Sphinx simulations which is a derivative of pyiron_atomistics.job.generic.GenericJob.
-    The functions in these modules are written in such the function names and attributes are very generic
-    (get_structure(), molecular_dynamics(), version) but the functions are written to handle Sphinx specific input and
-    output.
+    Class to setup and run Sphinx simulations which is a derivative
+    of pyiron_atomistics.job.generic.GenericJob. The functions in
+    these modules are written in such the function names and attributes
+    are very generic (get_structure(), molecular_dynamics(), version) but
+    the functions are written to handle Sphinx specific input and output.
```

Comment:
I improved this and a few other docstrings in 154e30f

> putting a new top-level group

Right now this is not really possible, but I agree it should eventually be enabled. If we agree to convert Input itself to a Group object, this will be pretty easy to add.

Reply:
Thanks. Let's discuss "Input is a Group" in tomorrow's meeting. I regard the docu issue fixed, otherwise.